### PR TITLE
Dmeerx 597 observation glucose labs

### PR DESCRIPTION
**Pull request by bmacnamara-work**
Labs for HBGM lab questionnaire.

### CODE REVIEW

File: fhirResourcesToLoad/o2jane_03__a1c.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "resourceType": "Observation",
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2jane_03__a1c.json:2]: Category is required

File: fhirResourcesToLoad/o2jane_03__observation_bloodGlucoseFasting.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "resourceType": "Observation",
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2jane_03__observation_bloodGlucoseFasting.json:2]: Need category

File: fhirResourcesToLoad/o2jane_03__observation_bloodGlucoseRandom.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "resourceType": "Observation",
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2jane_03__observation_bloodGlucoseRandom.json:2]: Need category

File: fhirResourcesToLoad/o2jane_03__observation_twoHourGlucoseTolerance.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "resourceType": "Observation",
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2jane_03__observation_twoHourGlucoseTolerance.json:2]: Add category

File: fhirResourcesToLoad/o2jane_04__service-request-a4217.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2jane_04__service-request-a4217.json:1]: This file should not be added as Priyank changed A4217 to DeviceRequest.

File: fhirResourcesToLoad/o2jane_03__observation_twoHourGlucoseTolerance.json (hunk 4 above)
Comment (bmacnamara-work) [reply fhirResourcesToLoad/o2jane_03__observation_twoHourGlucoseTolerance.json:2]: sure

File: fhirResourcesToLoad/o2jane_04__service-request-a4217.json (hunk 5 above)
Comment (bmacnamara-work) [reply fhirResourcesToLoad/o2jane_04__service-request-a4217.json:1]: sure

**Pull request merged**